ui/admin/app/routes/scopes/scope/targets.js

```diff
@@ -51,16 +51,30 @@ export default class ScopesScopeTargetsRoute extends Route {
   /**
    * Handle save.
    * @param {TargetModel} target
-   * @param {Event} e
+   * @param {TargetModel} egressEnabled
    */
   @action
   @loading
   @notifyError(({ message }) => message)
   @notifySuccess(({ isNew }) =>
     isNew ? 'notifications.create-success' : 'notifications.save-success'
   )
-  async save(target) {
-    await target.save();
+  async save(target, egressEnabled, updateDeprecatedWorkerFilter) {
+    //by default, when the deprecated worker_filter field is available,
+    //copy the filter value into egress filter field if egress filter is enabled
+    if (
+      updateDeprecatedWorkerFilter &&
```

Comment:
This is not a route level concern.  According to the RFC, saveWithToggles should accept the value of the toggles only.